lib/rubocop/cop/style/access_modifier_declarations.rb

```diff
@@ -96,6 +98,14 @@ def on_send(node)
 
         private
 
+        def access_modifier?(node)
+          maybe_access_modifier?(node) && node.access_modifier?
```

Comment:
This is fine, but shouldn't it be `Node#access_modifier?` itself that should be sped up instead of just this cop? I think just changing the order of the conditions in `rubocop-ast` should give you most of the speed up, but I may be wrong